File: behavioral/python/state/EURAccount.py
from AccountA import AccountA

class EURAccount(AccountA):
    def withdrawMoney(self, moneyAmount: int) -> None:
        print("Withdraw money of EUR account. " + str(moneyAmount) + "EUR")
    def depositMoney(self, moneyAmount: int) -> None:
        print("Deposit money of EUR account. " + str(moneyAmount) + "EUR")
    def getAccountName(self) ->str:
        return "EUR"

File: behavioral/jv/state/src/EURAccount.java
public class EURAccount implements AccountA{
    @Override
    public void withdrawMoney(int moneyAmount) {
        System.out.println("Withdraw money of EUR account. " + Integer.toString(moneyAmount) + "EUR");
    }

    @Override
    public void depositMoney(int moneyAmount) {
        System.out.println("Deposit money of EUR account. " + Integer.toString(moneyAmount) + "EUR");
    }

    @Override
    public String getAccountName() {
        return "EUR";
    }
}
